.github/actions/buildresources/action.yaml

```diff
@@ -59,6 +59,6 @@ runs:
       if: ${{ always() && inputs.jb-save == 'true'}}
       uses: actions/upload-artifact@v4
       with:
-        name: build-${{ inputs.os }}
+        name: build-${{ runner.os }}
```

Comment:
this is automatically set on all runners, so don't need to pass it as an input https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables